cmd/config.go

```diff
@@ -39,7 +39,7 @@ func addConfigFlags(flags *pflag.FlagSet) {
 	flags.String("recaptcha.host", "https://www.google.com", "use another host for ReCAPTCHA. recaptcha.net might be useful in China")
 	flags.String("recaptcha.key", "", "ReCaptcha site key")
 	flags.String("recaptcha.secret", "", "ReCaptcha secret")
-
+	flags.Uint("session.timeout", 2, "Set session timeout in hours (default is 2 hours)")
```

Comment:
Let's use `time.Duration` format instead. It is more readable than setting it as hours